code/modules/mob/mob_snapshot.dm

```diff
@@ -78,6 +78,9 @@
 	target.set_skin_tone(skin_tone)
 
 	for(var/limb_data in extra_limbs)
+		// For whatever reason, we already have a limb in this slot. Creating a new one would cause limb overwrite runtimes.
+		if(target.get_organ(limb_data))
```

Comment:
does this mean we can't overwrite the species tail now?

Reply:
I have added some handling for removing and replacing non-matching tails which should allow for this. Can't test because we don't have any tails that are available to tailed species at the moment.